conbench/entities/benchmark_result.py

```diff
@@ -280,6 +289,12 @@ def create(userres) -> "BenchmarkResult":
         result_data_for_db["hardware_id"] = hardware.id
         result_data_for_db["commit_id"] = commit.id if commit else None
         result_data_for_db["commit_repo_url"] = repo_url
+        result_data_for_db["history_fingerprint"] = generate_history_fingerprint(
+            case_id=case.id,
+            context_id=context.id,
```

Comment:
Agreed; I chose the UUIDs for speed today. But it would be nice to clean up/possibly denormalize these attributes, and then it would be easy to make it a function of the tag set.